strawberryfields/device.py

```diff
@@ -35,16 +34,21 @@ class DeviceSpec:
             - modes (int): number of modes supported by the target
             - compiler (list): list of supported compilers
             - gate_parameters (dict): parameters for the circuit gates
+
+        certificate (dict): the certificate of the device
```

Comment:
To match the description of `spec` above:
```suggestion
        certificate (dict, optional): dictionary representing the device certificate
```